bool status
string node_name
float32 node_data
